test/test_helpers.py

```diff
@@ -26,49 +26,60 @@
 
 
 def fake_script(test_case, name, content):
+    # raise unittest.SkipTest("fake_script not working on windows yet")
     if not hasattr(test_case, 'fake_script_path'):
         fake_script_path = tempfile.mkdtemp('-fake_script')
-        os.environ['PATH'] = '{}:{}'.format(fake_script_path, os.environ["PATH"])
+        test_case._old_path = os.environ["PATH"]
```

Comment:
Can't you just have `old_path` as a local var instead of polluting `test_case`? It should be accessible from `cleanup`...

Reply:
yep